app/Filament/Exports/ResultExporter.php

```diff
@@ -113,6 +113,16 @@ public static function getColumns(): array
                     return $record->download_latency_iqm;
                 })
                 ->enabledByDefault(false),
+            ExportColumn::make('downloaded_bytes')
```

Comment:
Since `downloaded_bytes` and `uploaded_bytes` are now columns you don't have to modify the state of the export column. You can use `ping` or `packet_loss` as a column examples.